services/inventory/grpc/agents_server.go

```diff
@@ -34,12 +35,35 @@ func NewAgentsServer(s *inventory.AgentsService) inventorypb.AgentsServer {
 	return &agentsServer{s}
 }
 
+var agentTypes = map[inventorypb.AgentType]models.AgentType{
```

Comment:
Don't we have that mapping already?

Reply:
No, we don't. Map we've made for PMM-5003 is different and we can't use it here.